assets/css/style.css

```diff
@@ -441,6 +440,210 @@ input {
   cursor: pointer;
 }
 
+/* Start Modal */
+.modal {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(6px);
+  position: fixed;
+  overflow: auto;
+  display: none;
+}
+
+.model-main {
+  height: auto;
+  min-height: auto;
+  left: 0;
+  top: 0;
+  background-color: #fff;
+  border-radius: 1rem;
+  padding: 1rem;
+  border: 1px solid #dfe1e6;
+  margin: 4rem 1rem 4rem 1rem;
+}
+
+.sub-div {
+  display: grid;
+  grid-template-rows: auto auto auto;
+  gap: 1.7rem;
+}
+
+.sub-div .img-sub-div {
+  width: auto;
+  height: auto;
+}
+
+.img-height {
+  width: 100%;
+  height: auto;
+  max-height: 586px;
+}
+
+.title-links-closeButton {
+  padding: 0;
+  margin: 0;
+  display: grid;
+  grid-template-rows: auto auto;
+  gap: 12px;
+}
+
+.title-links-closeButton-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100%;
+  margin: 0;
+}
+
+.p-information-row p {
+  line-height: 2rem;
+  font-size: 16px;
+  color: #344563;
+}
+
+.title-links-closeButton-title h2 {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 87%;
+  color: #091e42;
+  margin: 0;
+}
+
+.title-links-closeButton-title .fa-x {
+  font-size: 1rem;
+}
+
+.title-links-closeButton-title-links {
+  display: flex;
+  gap: 8px;
+}
+
+.information-row-three {
+  display: grid;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.linkAndButtons {
+  display: grid;
+  grid-template-rows: auto auto auto auto;
+  margin-top: 8px;
+}
+
+.fa-x-button {
+  margin-right: 24px;
+  font-weight: 600;
+  color: #344563;
+  font-size: 1.2rem !important;
+  cursor: pointer;
+}
+
+.tec-link {
+  margin-top: 8px;
+}
+
+.tec-links {
+  margin-top: 12px;
+  height: auto;
+  display: none;
+}
+
+.tec-linkss {
+  margin-top: 12px;
+  height: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.tec-linkss img {
+  width: 100%;
+  margin-top: 8px;
+}
+
+.SeeLive-SeeResources {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 0.5rem;
+  margin-top: 16px;
+}
+
+.botton-desktop-see {
+  width: 100%;
+  max-height: 48px;
+}
+
+.botton-desktop-see button {
+  width: 100%;
+  height: 48px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #396df2;
+  background-color: transparent;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  cursor: pointer;
+}
+
+.detail-1 {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.detail-2,
+.detail-3 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #6b778c;
+}
+
+.doks {
+  display: flex;
+  align-items: center;
+}
+
+/* tecnologies  */
+.technologies-desktop-component {
+  display: flex;
+  padding: 0;
+  gap: 8px;
+}
```

Comment:
Your design

Please kindly remove the extra placeholder as shown below
![image](https://github.com/WilfriedM01/portfolio/assets/78251927/088413da-0661-471a-aa91-fd7e703d5246)

Figma
![image](https://github.com/WilfriedM01/portfolio/assets/78251927/9d66e464-ccac-4d02-a420-a8658561856e)



[OPTIONAL] - It would be nice if we add CSS `flex-wrap: wrap` to fix the issue of the item `Ruby on rails`